### ISSUE TITLE
too many broken links to "/doxygen/" (Trac #2047)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2039]
There are approx. 200 broken links in the source tree (that I found with a simple `find|grep`). Should probably discuss moving the new doxygen output to "doxygen" as opposed to the current "cpp".

Migrated from https://code.icecube.wisc.edu/ticket/2047
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:44", 
    "description": "There are approx. 200 broken links in the source tree (that I found with a simple `find|grep`). Should probably discuss moving the new doxygen output to \"doxygen\" as opposed to the current \"cpp\".\n\n", 
    "reporter": "nega", 
    "cc": "nega, olivas", 
    "resolution": "fixed", 
    "_ts": "1550067284370534", 
    "component": "combo core", 
    "summary": "too many broken links to \"/doxygen/\"", 
    "priority": "normal", 
    "keywords": "documentation", 
    "time": "2017-06-29T18:02:42", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20170706T14:55:41: 
* nega changed status from "new" to "closed"
* nega commented:

> In changeset 2939:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2939"
> fix broken links to the doxygen docs
> 
> 1 commit enter. 200 broken links leave.
> or something.
> 
> this fixes most of the broken links to "/doxygen/".
> 
> there's still broken links. it would be a good idea to run all the docs
> through validator.w3.org and clean up those links and links to permenantly
> redirected urls.
> 
> see #2037
> closes #2039
> ```


* nega changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T23:44:46: nwhitehorn commented:

> In changeset 170014:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170014"
> fix broken links to the doxygen docs
> 
> 1 commit enter. 200 broken links leave.
> or something.
> 
> this fixes most of the broken links to "/doxygen/".
> 
> there's still broken links. it would be a good idea to run all the docs
> through validator.w3.org and clean up those links and links to permenantly
> redirected urls.
> 
> see #2037
> closes #2039
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:44: 
* olivas changed milestone from "July 2017" to ""
* olivas commented:

> Milestone July 2017 deleted
